packages/renderer/src/lib/deployments/DeploymentColumnAge.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+import type { DeploymentUI } from './DeploymentUI';
+
+export let object: DeploymentUI;
+</script>
+
+<div class="text-xs text-gray-500">
+  {object.age}
+</div>
```

Comment:
now it seems you can use your custom rendering / simple column

Reply:
Rebased on main to pick this up, and replaced Age and Namespace columns.